### DISCUSSION TITLE
Should show languages flow has translations for even if not part of workspace

### DISCUSSION THREAD

**Discussion by nicpottier** [nyaruka/floweditor#1155] [Ideas]
Right now editor shows the workspace languages as a set of "tabs" in the upper right to allow for translating a flow. That does not include other languages which may be present for the flow. (either from old languages or more likely because the flow was imported)

This can cause problems as we calculate dependencies based on all the translations for a flow, so a field can be in a state where it isn't used in any visible translations to the user but it can't be deleted because we view it as a dependency.

Feels to me like the editor should honor the translations that exist and make those available to be edited but perhaps with a different treatment. (ie, maybe a down arrow/drop down to the right of the workspace languages that shows other languages the flow has translations for and lets the user switch into them)

Seems like a natural consequence of this would be a way to delete some of these languages for a flow as well, so maybe the set of languages a flow is translated for is also a property of the flow and is editable at the flow level. (IE: I don't want this language available anymore, or I want to add this language)

**Comment by rowanseymour** [comment]
I think I like the idea of the editor allowing users to add/remove translations in any language regardless of which languages their workspace is configured to run flows in - whilst still making the workspace languages the defaults.
